packages/abstraxion-core/src/GranteeSignerClient.ts

```diff
@@ -93,7 +100,7 @@ export class GranteeSignerClient extends SigningCosmWasmClient {
   public async signAndBroadcast(
     signerAddress: string,
     messages: readonly EncodeObject[],
-    fee: StdFee | "auto" | number,
+    fee: StdFee | "auto" | number = "auto",
```

Comment:
We should keep this api inline with the cosmjs interface
```suggestion
    fee: StdFee | "auto" | number,
```